android/app/src/main/java/org/mozilla/magnet/db/History.java

```diff
@@ -0,0 +1,192 @@
+package org.mozilla.magnet.db;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.util.Log;
+
+import org.mozilla.magnet.db.Schema.HistoryEntry;
+
+import java.util.Date;
+import java.util.concurrent.TimeUnit;
+
+import static android.provider.BaseColumns._ID;
+import static org.mozilla.magnet.db.Schema.HistoryEntry.COLUMN_NAME_TIME_LAST_SEEN;
+import static org.mozilla.magnet.db.Schema.HistoryEntry.COLUMN_NAME_URL;
+import static org.mozilla.magnet.db.Schema.HistoryEntry.COLUMN_NAME_TIME_FIRST_SEEN;
+
+public class History extends SQLiteOpenHelper {
```

Comment:
I'm not sure I follow. I used [this tutorial](http://mobilesiri.com/android-sqlite-database-tutorial-using-android-studio/) to guide me .